specs/phase0/fork-choice.md

```diff
@@ -173,6 +180,17 @@ def get_ancestor(store: Store, root: Root, slot: Slot) -> Root:
         return root
 ```
 
+#### `calculate_committee_fraction`
+
+```python
+def calculate_committee_fraction(state: BeaconState, committee_percent: uint64) -> Gwei:
+    num_validators = len(get_active_validator_indices(state, get_current_epoch(state)))
+    avg_balance = get_total_active_balance(state) // num_validators
+    committee_size = num_validators // SLOTS_PER_EPOCH
+    committee_weight = committee_size * avg_balance
```

Comment:
```suggestion
       committee_weight = get_total_active_balance(state) // SLOTS_PER_EPOCH
```

Reply:
This function is also used to calculate the proposer boost, so this would be changing the semantics slightly.

I assume there's a reason why we use the `avg_balance` when calculating the boost currently? Maybe @adiasg or @fradamt know?

Reply:
I didn't understand this comment? this function only uses the product which cancels `num_validators`, that's my only innocent observation here.

Reply:
There's integer division in play which means they may not be entirely equivalent. It's probably a very minor difference but could cause a view split if we change it not on a hard fork boundary.